book/caveats.nim

```diff
@@ -0,0 +1,81 @@
+import nimib, nimibook
+
+
+nbInit
+nbUseNimibook
+
+nbText: """
+# Caveats
+There are some caveats when working with norm that you need to consider and strategies to work around them.
+
+## Managing data for Many-To-Many relationships
```

Comment:
I'm using title case in the docs, i.e. all words except for articles and prepositions start with caps: Managing Data for Many-to-Many Relationships.

Reply:
Done